pandas/core/generic.py

```diff
@@ -2339,6 +2339,8 @@ def select(self, crit, axis=0):
         """
         Return data corresponding to axis labels matching criteria
 
+        DEPRECATED: use df.loc[labels.map(crit)] to select via labels
```

Comment:
maybe `df.index` instead of `labels` ?

Reply:
changed, I put the labels because you can actually select on any axis (though it does default to 0)

Reply:
ah, yes, that was a good reason :-)